packages/react/src/hooks/useUIElementPositioning.ts

```diff
@@ -7,12 +7,21 @@ import {
 } from "@floating-ui/react";
 import { useEffect, useMemo } from "react";
 
+type UIElementPosition = {
```

Comment:
I'm gessing a ts change or sth that made this necessary?

Reply:
Yep, otherwise:

```
 boolean,
YousefED1 day ago
I'm gessing a ts change or sth that made this necessary?

The inferred type of 'useUIElementPositioning' cannot be named without a reference to '.pnpm/@floating-ui+react-dom@2.1.2_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/@floating-ui/react-dom'. This is likely not portable. A type annotation is necessary.
```

Triggered because I regenerated the node_modules so it upgraded TS